app/views/hyrax/embargoes/_list_expired_active_embargoes.html.erb

```diff
@@ -0,0 +1,44 @@
+<% if assets_with_expired_embargoes.empty? %>
```

Comment:
I think you may want to switch this to `if assets_with_expired_embaroes.present?` and switch the bodies of the `if` and `else` statements.